docs/docs/user/what-is-kpops.md

```diff
@@ -1,7 +1,26 @@
 # What is KPOps?
 
-KPOps is a CLI tool that allows deploying and destroying consecutive Kafka Streams applications, Producers, and Kafka 
-Connectors using an easy-to-read and write pipeline definition. KPOps comes with various abstractions that can simplify
-configuring multiple pipelines and steps within pipelines by sharing configuration between different applications,
-like producers or streaming applications. In addition, Kafka apps and its resources can be reset or cleaned up 
-again with a single command.
+With a couple easy commands in the shell and a [`pipeline.yaml`](#example) of under 30 lines, [KPOps](/) can not only [`deploy`](/user/references/cli-commands/#deploy) a pipeline[^1] to a Kubernetes cluster, but also [`reset`](/user/references/cli-commands/#reset), [`clean`](/user/references/cli-commands/#clean) or [`destroy`](/user/references/cli-commands/#destroy) it!
+[^1]:
+     A pipeline can consist of just consecutive [Kafka Streams applications](/user/references/components/#streamsapp) or it could also contain [Producers](/user/references/components/#producer) and/or [Kafka Connectors](/user/references/defaults/#kafkaconnector).
+
+## Key features
+
+- Deploy pipelines with minimal effort and without sacrificing configurability
+- Easy-to-read and write pipeline definitions
+- Reset or destroy pipelines gracefully
+- Reset or clean apps again with a single command
+- Set defaults to share settings between pipeline steps (or whole different pipelines!)
```

Comment:
Should we update the `Future` section of the readme with these points?

Reply:
Im not sure tbh, do you think they are better? If not, I could instead replace these with the points from the README.

Reply:
Actually not! I think both of the sections provide good insights about KPOps keept it like that